lib/api/repository/user_repository.dart

```diff
@@ -0,0 +1,33 @@
+import 'package:injectable/injectable.dart';
+import 'package:survey/api/request/login_request.dart';
+import 'package:survey/api/response/login_response.dart';
+import 'package:survey/api/service/user_service.dart';
+import 'package:survey/flavors.dart';
+
+const String _grantTypePassword = 'password';
+
+abstract class UserRepository {
```

Comment:
same, this one should be AuthRepository or NonAuthenticatedRepository, etc, not User_